final_assignment/marius_ilau/Main.py

```diff
@@ -0,0 +1,33 @@
+import subprocess
+import os
+import urllib
+import argparse
+
+
+def before_install():
```

Comment:
nu stii ca in before_install o sa fie un download, poate sa fie un run_command sau alt ceva